modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc

```diff
@@ -120,23 +120,23 @@ For more information about the privilege, see xref:authentication-authorization/
 
 .Drop a database and the local database alias targeting it
 ====
-.Delete a database with local database aliases targeting it
-====
-The following example assumes that you have a database that is the target of a local database alias:
+The following example creates a database `movies` and a local database alias `films` targeting it:
+
 [source, cypher]
 ----
 CREATE DATABASE movies
 CREATE ALIAS films FOR DATABASE movies
 ----
 
-Then, both the database alias `films` and the database `movies` can be dropped with the following:
+Then, the database `movies` and the local database alias `films` can be dropped using the following command:
+
 [source, cypher]
 ----
 DROP DATABASE movies CASCADE ALIASES
 ----
+
 ====
-====
-The option `RESTRICT` explicitly requests the default behavior of the command.
+//The option `RESTRICT` explicitly requests the default behavior of the command.
```

Comment:
Here it feels more like we should mention it somewhere, as this page have the syntax description at the top... my comment was on the composite page